docs/modding/suitmodding/addinglocals.mdx

```diff
@@ -0,0 +1,666 @@
+3. <Tooltip text={<strong>Repeat the Conversion</strong>} tooltip="Using the Localisation Hash calculator" />: Use <Tooltip text={<strong>Text Tools</strong>} tooltip="A tab in GlacierKit that provides tools for converting text and hashes for game modifications." /> again to convert any new <Tooltip text={<strong>strings</strong>} tooltip= "A key that links to text in the LOCR File" /> into <Tooltip text={<strong>decimals</strong>} tooltip="A string converted into a numeral form" />.
+4. **Update the** <Tooltip text={<strong>manifest</strong>} tooltip="A file in your mod folder that holds important information and can execute code" />: Add the new language entries, keeping everything consistent.
+
+Example of every aspect from this tutorial applied:
+```json
+"localisationOverrides": {
+    "yourLocrHASH": {
+        "english": {
+            "yourconverteddecimal": "New Suit Name",
+            "yourotherconverteddecimal": "New Suit Description/or Other New Suit name"
+        }
+    },
+    "otherLocrHASH": {
+        "english": {
+            "somenewconverteddecimal": "More New Suit Name",
+            "someotherconverteddecimal": "More New Suit Description/or Other More New Suit name"
+        },
+        "spanish": {
+            "somenewconverteddecimal": "Más nombre de traje nuevo",
+            "someotherconverteddecimal": "Más descripción del traje nuevo/u otro nombre del traje más nuevo"
+        }
+    }
+}
+```
+
+*Remember to keep your* <Tooltip text={<strong>manifest</strong>} tooltip="A file in your mod folder that holds important information and can execute code" /> *clean and organized to avoid errors.*
+
+---
+
+## Advanced Search Guide
+[[Return to tutorial]](#3-confirm-your-locr)
+
+*When you need to find text for something other than suits, use the* <Tooltip text={<strong>Advanced Search Tab</strong>} tooltip="A tab in GlacierKit for performing extended searches that go beyond the 'game contents' tab search abilities. It has a magnifying glass icon and, is like all other tabs, on the left side." />. 
+
+Start with:
+
+1. Clicking on the <Tooltip text={<strong>Advanced Search Tab</strong>} tooltip="A tab in GlacierKit for performing extended searches. Located on the left." />
+
+2. **Set Your Search Parameters**:
+   - Check only the `chunk0` box.
+    - Tick the **"localisation"** option.
+
+    :::note
+        - *`Chunk0` holds most of the text data, making the search more efficient.*
+
+        - The latter will refine your search to only localisation-related text files.
+    
+
+    :::
+
+
+3. **Try Multiple Case Variations**:
+
+    *Advanced search is* ***case sensitive*** *so try multiple type of search operations at the* ***same time***
+
+    Search using different case-sensitive variations of your text. Examples:
+     - "Blood Money"
+     - "blood money"
+     - "BLOOD MONEY"
+     - "bloodmoney"
```

Comment:
Advanced search uses regex, `(?i)blood money` will search for all variations of `blood money` without being case-sensitive